packages/frontend/src/components/DashboardTiles/TileForms/AddChartTilesModal.tsx

```diff
@@ -1,71 +1,96 @@
-import { Button, Classes, Dialog, Icon, Intent } from '@blueprintjs/core';
 import {
     Dashboard,
     DashboardTileTypes,
     defaultTileSize,
 } from '@lightdash/common';
-import React, { FC, useMemo } from 'react';
-import { useForm } from 'react-hook-form';
+import {
+    Button,
+    Group,
+    Modal,
+    MultiSelect,
+    Stack,
+    Text,
+    Title,
+    Tooltip,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { uuid4 } from '@sentry/utils';
```

Comment:
weird that we import this from sentry. is there a reason behind it `import { v4 as uuid4 } from 'uuid';`?

Reply:
I let imports resolve themselves so I'll fix this